exercises/rotational-cipher/example.py

```diff
@@ -0,0 +1,12 @@
+def rotational_cipher(message, key):
```

Comment:
The convention is to use nouns for classes and verbs for functions. So in this case I would prefer `rotate()`.